### PR TITLE
Adding commit message to zipDeploy payload 

### PR DESCRIPTION
**Pull request by BALAGA-GAYATRI**
Resolving https://github.com/Azure/webapps-deploy/issues/118

Not using action.yml for taking the input of commit-message

### CODE REVIEW

File: src/actionparameters.ts [hunk 1]
```diff
@@ -1,135 +1,140 @@
...
+    public static getActionParams(endpoint?: IAuthorizer) {
+        if (!this.actionparams) {
+            this.actionparams = new ActionParameters(!!endpoint ? endpoint : null);
+        }
+        
+        return this.actionparams;
+    }
+    public get appName() {
+        return this._appName;
+    }
```
Comment (kaverma) [inline src/actionparameters.ts:58]: can you please revert this file, if there is no change?

**Comment by kaverma** [comment]
Please add tests also for this change

File: lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js [hunk 2]
```diff
@@ -75,5 +76,14 @@ class WebAppDeploymentProvider extends BaseWebAppDeploymentProvider_1.BaseWebApp
             }
         });
     }
+    UpdateDeploymentStatus(isDeploymentSuccess) {
```
Comment (kaverma) [inline lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js:79]: nit: newline before the method starts

Comment (kaverma) [inline lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js:79]: Why is this method not present in the typescript file?

**Review by kaverma — CHANGES_REQUESTED** [review]
Left few comments, please have a look.

File: src/actionparameters.ts [hunk 3]
```diff
@@ -1,135 +1,140 @@
...
+
+    private constructor(endpoint: IAuthorizer) {
+        this._publishProfileContent = core.getInput('publish-profile');
+        this._appName = core.getInput('app-name');
+        this._slotName = core.getInput('slot-name');
+        this._packageInput = core.getInput('package');
+        this._images = core.getInput('images');
+        this._multiContainerConfigFile = core.getInput('configuration-file');
+        this._startupCommand = core.getInput('startup-command');
+        this._commitMessage = github.context.payload.head_commit.message;
```
Comment (AmrutaKawade) [inline src/actionparameters.ts:45]: add null check. remove formatting if anything applied on this file diff will be proper

Comment (BALAGA-GAYATRI) [reply src/actionparameters.ts:47]: Haven't made any changes to formatting. Can you try removing white space in diff settings button. Its working temporARLY

File: lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js (hunk 2 above)
Comment (BALAGA-GAYATRI) [reply lib/DeploymentProvider/Providers/WebAppDeploymentProvider.js:79]: It is there. https://github.com/Azure/webapps-deploy/blob/a98be40ccf5d3017a21681529ff244f96d597fca/src/DeploymentProvider/Providers/WebAppDeploymentProvider.ts#L72

**Review by kaverma — COMMENTED** [review]
Want to validate ChatOps notifications

File: package.json [hunk 4]
```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "@actions/core": "^1.2.6",
+    "@actions/github": "^4.0.0",
```
Comment (kaverma) [inline package.json:35]: this is a new dependency needed now?

**Review by kaverma — APPROVED** [review]

**Pull request merged**